spec/adaptors/operation_contract_spec.rb

```diff
@@ -40,7 +40,7 @@ def operation_paths
     def resource_attributes_for(resource_klass:)
       return [] if resource_klass < LedgerSync::Error
 
-      resource_klass.attributes | [:ledger_id]
+      resource_klass.attributes.keys | [:ledger_id]
```

Comment:
Should this be `|` or `||`?

Reply:
It's an intersection, so adding the `:ledger_id` key if it doesn't exist in the keys already.